### PR TITLE
Add support for removing ih8sn

### PR DESCRIPTION
**Pull request by althafvly**
- Flashing again will remove ih8sn

### CODE REVIEW

File: META-INF/com/google/android/update-binary [hunk 1]
```diff
@@ -108,6 +108,14 @@ else
 error_mounting "$SYSTEM_MNT"
 fi
 
+if [[ `ls ${SYSTEM_MNT}/system/bin/ih8sn 2> /dev/null` ]]; then
+  ui_print "Removing existing ih8sn files"
+  find ${SYSTEM_MNT}/system | grep ih8sn | xargs rm
```
Comment (luk1337) [inline META-INF/com/google/android/update-binary:113]: `find ${SYSTEM_MNT}/system -name *ih8sn* -delete`

Comment (luk1337) [reply META-INF/com/google/android/update-binary:113]: similar suggestion to other places with `find | grep | xargs`

Comment (althafvly) [reply META-INF/com/google/android/update-binary]: done

**Comment by althafvly** [comment]
ignore that. accident

**Pull request closed without merging**